sympy/physics/vector/tests/test_printing.py

```diff
@@ -4,183 +4,202 @@
+a/b * N.x + (c+b)/a * N.y + c**2/b * N.z
 
-def unicode_vpretty(expr):
-    return vpprint(expr, use_unicode=True, wrap_line=False)
+Dyadic:
+a ** 2 * (N.x | N.y) + b * (N.y | N.y) + c * sin(alpha) * (N.z | N.y)
+alpha * (N.x | N.x) + sin(omega) * (N.y | N.z) + alpha * beta * (N.z | N.x)
 
-def test_latex_printer():
-    r = Function('r')('t')
-    assert VectorLatexPrinter().doprint(r ** 2) == "r^{2}"
 
-def test_vector_pretty_print():
+VectorLatexPrinter
+
+(a ** 2 + b / c) * A.x + sqrt(d) * A.y + cos(omega) * A.z
+theta * A.x + omega * omega * A.y + (q * alpha) * A.z
+sin(theta1) * A.x + cos(phi1) * cos(phi2) * A.y + cos(theta1 + phi3) * A.z
+(a ** 2 + b / c) * N.x + sqrt(d) * N.y + cos(omega) * N.z
+
+Custom unit vectors:
+(a ** 2 + b / c) * _N.x + sqrt(d) * _N.y + cos(omega) * _N.z
+
+Functions of time (dynamicsymbols):
+omega.diff() * N.x
+omega.diff() ** alpha * N.x
+
+Dyadic:
+a ** 2 * (N.x | N.y) + b * (N.y | N.y) + c * sin(alpha) * (N.z | N.y)
+alpha * (N.x | N.x) + sin(omega) * (N.y | N.z) + alpha * beta * (N.z | N.x)
+
+"""
+
 
-    # TODO : The unit vectors should print with subscripts but they just
-    # print as `n_x` instead of making `x` a subscript with unicode.
+def vpretty(expr):
+    """ASCII pretty-printing"""
+    return vpprint(expr, use_unicode=False, wrap_line=False)
+
+
+def uvpretty(expr):
+    """Unicode pretty-printing"""
+    return vpprint(expr, use_unicode=True, wrap_line=False)
 
-    # TODO : The pretty print division does not print correctly here:
-    # w = alpha * N.x + sin(omega) * N.y + alpha / beta * N.z
 
-    expected = """\
+def test_vpretty_basic():
+    """ASCII, Unicode basic testing"""
+
+    expr = a ** 2 * N.x + b * N.y + c * sin(alpha) * N.z
+    ascii_str = """\
  2
 a  n_x + b n_y + c*sin(alpha) n_z\
 """
-    uexpected = u("""\
+    ucode_str = u("""\
  2
-a  n_x + b n_y + c⋅sin(α) n_z\
+a  nₓ + b n_y + c⋅sin(α) n_z\
 """)
+    assert vpretty(expr) == ascii_str
+    assert uvpretty(expr) == ucode_str
 
-    assert ascii_vpretty(v) == expected
-    assert unicode_vpretty(v) == uexpected
-
-    expected = u('alpha n_x + sin(omega) n_y + alpha*beta n_z')
-    uexpected = u('α n_x + sin(ω) n_y + α⋅β n_z')
-
-    assert ascii_vpretty(w) == expected
-    assert unicode_vpretty(w) == uexpected
+    expr = alpha * N.x + sin(omega) * N.y + alpha * beta * N.z
+    ascii_str = """\
+alpha n_x + sin(omega) n_y + alpha*beta n_z\
+"""
+    ucode_str = u("""\
+α nₓ + sin(ω) n_y + α⋅β n_z\
+""")
+    assert vpretty(expr) == ascii_str
+    assert uvpretty(expr) == ucode_str
 
-    expected = """\
+    expr = a/b * N.x + (c+b)/a * N.y + c**2/b * N.z
+    ascii_str = """\
                      2
 a       b + c       c
 - n_x + ----- n_y + -- n_z
 b         a         b\
 """
-    uexpected = u("""\
-                     2
-a       b + c       c
-─ n_x + ───── n_y + ── n_z
-b         a         b\
+    ucode_str = u("""\
+                    2
+a      b + c       c
+─ nₓ + ───── n_y + ── n_z
```

Comment:
Why is subscript `x` in unicode, while `y` and `z` are not?

Reply:
@gxyd Because they are originally intended for phonetic characters.

Reply:
Unicode does not have subscripts for all characters. Since, y and z are not available, this should revert the subscript x to `_x`, or something better.